e2e/typetest.tsx

```diff
@@ -8,6 +8,12 @@ function expectType<T>(_value: T) {}
 
 // next/link
 
+// Links with string hrefs
+<Link href="/" />;
+<Link href="/foos/[foo]" />;
```

Comment:
@MariaSolOs We shouldn't accept this string right? Without the query argument, NextJS won't properly route this path

Reply:
You're completely right! Thanks for catching that!